ega-charts/localega/templates/dataedge-deploy.yaml

```diff
@@ -30,12 +30,25 @@ spec:
         env:
         - name: SPRING_PROFILES_ACTIVE
           value: no-oss
+        - name: FILEDATABASE_SERVERS
+          value: {{ if .Values.config.filedatabase_host }}{{ .Values.config.filedatabase_host }}:{{ .Values.config.filedatabase_port }}{{ else }}{{ template "localega.fullname" . }}-filedatabase:{{ .Values.filedatabase.servicePort }}{{ end }}
+        - name: KEYS_SERVERS
+          value: {{ if .Values.config.keyserver_host }}{{ .Values.config.keyserver_host }}:{{ .Values.config.keyserver_port }}{{ else }}{{ template "localega.fullname" . }}-keys:{{ .Values.keys.servicePort }}{{ end }}
+        - name: RES_SERVERS
+          value: {{ if .Values.config.res_host }}{{ .Values.config.res_host }}:{{ .Values.config.res_port }}{{ else }}{{ template "localega.fullname" . }}-res:{{ .Values.res.servicePort }}{{ end }}
```

Comment:
Might be good to add `quote` to the new ENV vars too

Reply:
Done.